attributes/default.rb

```diff
@@ -0,0 +1,6 @@
+#service name definition
+default['w32time']['service'] = 'w32time'
```

Comment:
I think we can hardcode this instead of having an attribute for a value that'll "never" change